src/Illuminate/Support/helpers.php

```diff
@@ -523,3 +523,21 @@ function with($value, ?callable $callback = null)
         return is_null($callback) ? $value : $callback($value);
     }
 }
+
+if (! function_exists('to_english_numbers')) {
+    /**
+     * Converting Persian/Arabic digits to English
+     * @param  mixed  $value
+     * @return string
+     */
+    function to_english_numbers($value): string
+    {
+        return strtr($value, [
+            '۰' => '0', '۱' => '1', '۲' => '2', '۳' => '3', '۴' => '4',
+            '۵' => '5', '۶' => '6', '۷' => '7', '۸' => '8', '۹' => '9',
+            '٠' => '0', '١' => '1', '٢' => '2', '٣' => '3', '٤' => '4',
+            '٥' => '5', '٦' => '6', '٧' => '7', '٨' => '8', '٩' => '9',
+        ]);
```

Comment:
You could just do
```suggestion
        new NumberFormatter('ar-u-nu-latn', NumberFormatter::DECIMAL);
        return $numberFormatter->parse($value);
```
and have a safe, native way to handle this